util/generate-addresses.ts

```diff
@@ -12,7 +11,7 @@ class App {
     const wallet = await WalletHelper.restore(code);
     wallet.initialize();
 
-    const addresses = await wallet.getAddresses(Config.wallet.addressCount);
+    const addresses = await wallet.getAddresses(0, Config.wallet.addressCount);
```

Comment:
we need to not forget to adapt this if needed

Reply:
Yes